### PR TITLE
251 gview add tutorial pictures with explanations when gview is first opened when gviewini is not found

### PR DESCRIPTION
**Pull request by rzaharia**

### CODE REVIEW

File: GViewCore/src/App/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1 @@
-target_sources(GViewCore PRIVATE ErrorDialog.cpp GViewApp.cpp FileWindow.cpp FileWindowProperties.cpp Instance.cpp SelectTypeDialog.cpp)
\ No newline at end of file
+target_sources(GViewCore PRIVATE ErrorDialog.cpp GViewApp.cpp FileWindow.cpp FileWindowProperties.cpp Instance.cpp SelectTypeDialog.cpp TutorialWindow.cpp)
```
Comment (xTachyon) [inline GViewCore/src/App/CMakeLists.txt:1]: surse pe linii diferite?

File: GViewCore/src/App/Instance.cpp [hunk 2]
```diff
@@ -11,8 +11,7 @@ constexpr uint32 MIN_CACHE_SIZE        = 0x10000;  // 64 K
 constexpr uint32 GENERIC_PLUGINS_CMDID = 40000000;
 constexpr uint32 GENERIC_PLUGINS_FRAME = 100;
 
-struct _MenuCommand_
-{
+struct _MenuCommand_ {
```
Comment (xTachyon) [inline GViewCore/src/App/Instance.cpp:14]: undefined

**Review by xTachyon — APPROVED** [review]

File: GViewCore/src/App/CMakeLists.txt (hunk 1 above)
Comment (rzaharia) [reply GViewCore/src/App/CMakeLists.txt]: fixed

File: GViewCore/src/App/Instance.cpp (hunk 2 above)
Comment (rzaharia) [reply GViewCore/src/App/Instance.cpp]: fixed in 95d3c2c669b3faf3d44b106626e4ec8aea86d74b

**Pull request merged**